### DISCUSSION TITLE
Package name

### DISCUSSION THREAD

**Discussion by kuhnroyal** [cfug/diox#26] [Ideas]
I wonder if the new package name should be something different than `dio3` - I personally have no idea whey there is a `3`. in the name, because we plan to release version 4, right? :) 

Maybe something more generic would make sense?

**Comment by AlexV525** [comment]
It's just something like Web 3. :D

**Reply by kuhnroyal** [reply]
What about something more generic `diox` or something?

**Reply by AlexV525** [reply]
cc @chenglu

**Comment by chenglu** [comment]
+1 to Alex, we decided to launch this project mainly because `dio` was not maintained well, and the main reason is that the `dio` is too centralized. So we called this new project as `dio3`, with the 3 standing for the Web 3 ;-D

Also, `dio3` is kind of the internal project name, for the new `dio`, our expectation is to ensure continuous updates and maintenance of the project, as well as focus on reducing migration costs for users.

If the public name will be a more generic one, I think that's a perfect idea!

**Comment by AlexV525** [comment]
We'll take `diox` as the new package name after we discussed it offline. Thanks @kuhnroyal

**Reply by chenglu** [reply]
Do we need also update the package name on pub.dev?

**Reply by AlexV525** [reply]
> Do we need also update the package name on pub.dev?

Yes, it'll require uploading a new package instead.

**Comment by AlexV525** [comment]
The package is now occupied by us.

https://pub.dev/packages/diox
